website/sidebars.js

```diff
@@ -6,30 +6,55 @@
  */
 
 module.exports = {
-  docs: {
-    Docusaurus: ['introduction', 'design-principles', 'contributing'],
-    'Getting Started': ['installation', 'configuration'],
-    Guides: [
-      'creating-pages',
-      'styling-layout',
-      'static-assets',
-      {
-        Docs: ['docs', 'markdown-features', 'versioning'],
-      },
-      'blog',
-      'search',
-      'deployment',
-      'migrating-from-v1-to-v2',
-    ],
-    'Advanced Guides': ['using-plugins', 'using-themes', 'presets'],
-    'API Reference': [
-      'cli',
-      'docusaurus-core',
-      'docusaurus.config.js',
-      'lifecycle-apis',
-      'theme-classic',
-    ],
-  },
+  docs: [
+    {
+      type: 'category',
+      label: 'Docusaurus',
+      collapsed: false,
+      items: ['introduction', 'design-principles', 'contributing'],
+    },
+    {
+      type: 'category',
+      label: 'Getting Started',
+      collapsed: true,
+      items: ['installation', 'configuration'],
+    },
+    {
+      type: 'category',
+      label: 'Guides',
+      collapsed: true,
+      items: [
+        'creating-pages',
+        'styling-layout',
+        'static-assets',
+        {
+          Docs: ['docs', 'markdown-features', 'versioning'],
+        },
+        'blog',
+        'search',
+        'deployment',
+        'migrating-from-v1-to-v2',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Advanced Guides',
+      collapsed: true,
+      items: ['using-plugins', 'using-themes', 'presets'],
+    },
+    {
+      type: 'category',
+      label: 'API Reference',
+      collapsed: true,
+      items: [
+        'cli',
+        'docusaurus-core',
+        'docusaurus.config.js',
+        'lifecycle-apis',
+        'theme-classic',
+      ],
+    },
+  ],
```

Comment:
@lex111 is this what you had in mind?

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/3806031/80430922-e66ae280-88a4-11ea-9a65-a468dacd4c60.png">

Reply:
@jsjoeio Yes, thanks, this is exactly what I had in mind, however, to achieve this it is necessary to change the sidebar structure (use an array instead of an object), but ok.

Hmm, should we highlight expanded by default category? In my understanding, we need to highlight the category in which we are currently (based on the current path).

For example, I am on the Guides -> Docs page, everything is fine, two related items (Guides and Docs-Introduction) are highlighted in green. 
But at the same time, we also see that the top-level item with Docusaurus is also highlighted as activated. I don’t know for sure, but it seems that similar behavior could lead people astray so we _don’t_ need to highlight the category of sidebar, which are expanded by default.

Any objections? cc @yangshun 

![image](https://user-images.githubusercontent.com/4408379/80472801-e4a92980-894d-11ea-814d-66a70a08b818.png)

Reply:
We should only highlight the active one. Opening/close doesn't result in highlighting.

Reply:
I see! 

Okay, I assume that's part of this work then - if you want to request changes on this PR, then I get started and ping you when it's ready for re-review @lex111